Makefile

```diff
@@ -7,12 +7,19 @@ $(error $(CONFIG_FILE) not found. See $(CONFIG_FILE).example.)
 endif
 include $(CONFIG_FILE)
 
+# Rectify input parameters
+ifeq ($(CPU_ONLY),1)
+  USE_CUDNN=0
+endif
```

Comment:
This can go in its own commit/PR, and please follow existing style (e.g., use space-padded `:=` assignment).